app/views/admin/users/show/flags/_index.html.erb

```diff
@@ -1,43 +1,46 @@
-          <% end %>
-        </span>
-        <div class="flex-1">
-          <h3 class="crayons-subtitle-2 mb-1">
-            <%= link_to flag.reactable.path, class: "c-link" do %>
-              <% if flag.reactable_type == "Article" %>
-                <%= t("views.admin.users.flags.type.article", title: flag.reactable.title) %>
-              <% elsif flag.reactable_type == "User" %>
-                <%= t("views.admin.users.flags.type.profile") %>
-              <% elsif flag.reactable_type == "Comment" %>
-                <%= t("views.admin.users.flags.type.comment", title: flag.reactable.title) %>
-              <% end %>
-            <% end %>
-          </h3>
-          <p class="fs-s color-secondary">
-            <%= t("views.admin.users.flags.origin_html", user: link_to(flag.user.username, admin_user_path(flag.user_id), class: "c-link c-link--branded")) %>
-          </p>
-        </div>
-        <time datetime="<%= flag.created_at&.strftime("%Y-%m-%dT%H:%M:%S%z") %>" class="color-secondary fs-s shrink-0" title="<%= l(flag.created_at, format: :admin_user) if flag.created_at %>">
-          <%= l(flag.created_at, format: :short_with_yy) %>
-        </time>
-      </article>
-    <% end %>
+<div class="flex justify-between gap-10">
+  <div class="flex flex-col flex-1">
+    <h2 class="crayons-subtitle-2 px-1"><%= t("views.admin.users.priviliged_actions.title") %></h2>
+    <p class="crayons-subtitle-3 fw-normal color-secondary px-1 mt-1"><%= t("views.admin.users.priviliged_actions.description") %></p>
   </div>
-<% else %>
-  <div class="align-center flex flex-col justify-center my-auto py-8">
-    <h3 class="crayons-subtitle-2 mb-2"><%= t("views.admin.users.flags.empty1", user: @user.name) %></h3>
-    <p class="color-secondary"><%= t("views.admin.users.flags.empty2", user: @user.name) %></p>
+  <div class="flex items-start gap-1">
+    <span class="crayons-card crayons-card--secondary px-3 py-1 flex gap-2 items-center" title="<%= t("views.moderations.actions.vomit") %>">
+      <%= crayons_icon_tag("twemoji/suspicious", native: true, width: 16, height: 16) %>
+      <span class="fs-s fw-medium lh-base"><%= countable_flags %></span>
+    </span>
+
+    <span class="crayons-card crayons-card--secondary px-3 py-1 ml-3 flex gap-2 items-center" title="<%= t("views.admin.users.flags.total_score") %>">
+      <%= crayons_icon_tag("analytics", native: true, width: 16, height: 16) %>
+      <span class="fs-s fw-medium lh-base"><%= score %></span>
+    </span>
   </div>
-<% end %>
+</div>
+
+<div id="reaction-content" class="flex flex-col gap-3 px-1 pt-6" style="overflow: auto; height: 406px;">
+  <% if user_vomit_reactions.present? %>
+    <% user_vomit_reactions.each do |vomit_reaction| %>
+      <%= render "admin/shared/flag_reaction_item", vomit_reaction: vomit_reaction, text_section: "users" %>
+      <hr id="js__reaction__div__hr__<%= vomit_reaction.id %>" class="w-100 hr-no-margins">
```

Comment:
I know that you didn't add this right now but this ID has me so confused  😅

Reply:
@Ridhwana I do understand this, I just kept it like this considering that it has been used like this in other places too.
Honestly, even I don't know what the exact convention is for naming it, just wanted to keep it consistent with other similar code.